ocs/fsimpl/src/main/java/org/sdo/iotplatformsdk/ocs/fsimpl/fs/FsPropertiesLoader.java

```diff
@@ -3,10 +3,14 @@
 
 package org.sdo.iotplatformsdk.ocs.fsimpl.fs;
 
-import java.io.FileReader;
-import java.io.IOException;
-import java.util.Properties;
 import java.util.concurrent.ConcurrentHashMap;
+import org.apache.commons.configuration2.EnvironmentConfiguration;
```

Comment:
Is there a possibility to create a common class for reading the properties and then extending that class in each of the services?

Reply:
The 3 components OPS, OCS and To0Scheduler are tied by beans defined in common/rest module as per the REST contract. So the common class could go there, and the properties loader classes would extend/implement it.
However, it is not mandatory for each component  to handle properties in the same manner, and it is not a part of any REST contract between the components, which is why these are separated out for each component, incidentally having the same class-level contents in them.